opennms-webapp-rest/src/test/resources/v1/acks.json

```diff
@@ -7,7 +7,7 @@
             "ackTime": 1437073152156, 
             "ackType": "UNSPECIFIED", 
             "ackUser": "admin", 
-            "id": 61, 
+            "id": 62,
```

Comment:
Hmm?

Reply:
this is a hard coded database id (I know - not very nice). It gets generated by https://github.com/OpenNMS/opennms/blob/foundation-2018/opennms-dao/src/main/java/org/opennms/netmgt/dao/DatabasePopulator.java . Since I added the creation of a CdpLink the sequence is one higher afterwards...